magenta/protobuf/music.proto

```diff
@@ -0,0 +1,125 @@
+syntax = "proto3";
+
+package tensorflow.magenta;
+
+// A message containing a symbolic music sequence. The design is largely
+// based on MIDI but it should be able to represent any music sequence.
+// For details see https://www.midi.org/specifications.
+message NoteSequence {
+  // NOTE: The first three fields are duplicated from SequenceFile to allow
+  // creation of portable SSTables of Sequence protos.
+  // Unique id. See data_utils.h GenerateSequenceId().
+  string id = 1;
+  // The path of the file relative to the root of the collection.
+  string filename = 2;
+  // The collection from which the file comes. This can be shorthand e.g.
+  // "bach". One purpose is to allow for easy selection of all or some files
+  // from a particular source.
+  string collection = 3;
+
+  // MIDI ticks per beat, also known as resolution or PPQ ("parts per quarter").
+  // There is no widely-used default. 220 is assumed per the choice made in
+  // third_party/py/pretty_midi.
```

Comment:
The last sentence needs to be rewritten. "If unspecified, 220 is assumed."

Not that this is the right place to complain, but 220 is a weird default value. Why not make triplets work perfectly? I'd vote for 60.